docs/deep-dives/mobile/first-run-experiments.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: mobile-first-run-experiments
+title: First Run Experiments
+slug: /mobile-first-run-experiments
+---
+
+First run experiments are incredibly important to the experimentation ecosystem, but their functionality comes with some key differences from standard experiments. It is _highly_ advised that experiment owners read through this documentation before writing first run experiments.
+
+:::note
+
+The concept of first run experiments is currently only supported on mobile applications, such as Firefox for Android/iOS and Focus.
```

Comment:
Could we link to docs on our capabilities for Desktop? Because we do support something functionally equivalent there

Reply:
Oh where is our desktop documentation?

Reply:
I assumed we had some when I wrote that comment... But now all I can find is [this](https://experimenter.info/desktop-feature-api/#can-i-use-this) which claims we support it for desktop

Reply:
Oh so maybe I just take this whole note out then

Reply:
Or reword it to explain that this particular set of docs is only pertaining to our mobile applications